sys/syz-extract/netbsd.go

```diff
@@ -23,14 +23,28 @@ func (*netbsd) prepare(sourcedir string, build bool, arches []string) error {
 	return nil
 }
 
-func (*netbsd) prepareArch(arch *Arch) error {
-	if err := os.Symlink(filepath.Join(arch.sourceDir, "sys", "arch", "amd64", "include"),
-		filepath.Join(arch.buildDir, "machine")); err != nil {
+func machineInclude(arch *Arch, machine string) string {
+	return filepath.Join(arch.sourceDir, "sys", "arch", machine, "include")
+}
+
+func machineLink(arch *Arch, machine string, dest string) error {
+	if err := os.Symlink(machineInclude(arch, machine),
+	    filepath.Join(arch.buildDir, dest)); err != nil {
 		return fmt.Errorf("failed to create link: %v", err)
 	}
-	if err := os.Symlink(filepath.Join(arch.sourceDir, "sys", "arch", "x86", "include"),
-		filepath.Join(arch.buildDir, "x86")); err != nil {
-		return fmt.Errorf("failed to create link: %v", err)
+	return nil;
+}
+
+func (*netbsd) prepareArch(arch *Arch) error {
+	links := [3][2]string {
```

Comment:
It is more idiomatic in Go and better for future changes to use `[]` instead of `[3]`. It will create a slice instead of an array with initial size deduced from the initializer.